earlydecoder/stacks/load_deploy.go

```diff
@@ -23,24 +21,29 @@ func loadDeployFromFile(file *hcl.File, ds *decodedStack) hcl.Diagnostics {
 	for _, block := range content.Blocks {
 		switch block.Type {
 		case "deployment":
-			content, _, contentDiags := block.Body.PartialContent(deploymentSchema)
-			diags = append(diags, contentDiags...)
+			// This produces incorrect diagnostics because it doesn't parse
+			// the HCL object correctly
```

Comment:
Should we just remove the commented-out code then? 🤔
Or is there a follow-up solution to making this work?

Reply:
Was left for Daniel to review. Removed

Reply:
I'm currently looking into that and am fighting some references

Reply:
I merged the other PR and rebased this one to fix the conflicts